packages/web/src/_page-tests/experience/__snapshots__/typography.test.tsx.snap

```diff
@@ -3093,7 +3102,7 @@ exports[`Experience/Typography renders 1`] = `
                     }
                   }
                 >
-                  typography.h4Example
+                  Subtitle / larger text that is one size larger than the body, lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.
```

Comment:
Should there be a break between the text and the lorem ipsum text?

Reply:
sounds like a good improvement